pages/shared/syncHelpers.js

```diff
@@ -229,3 +233,57 @@ module.exports.gitStatus = function(locals, callback) {
         statusStage1();
     });
 };
+
+module.exports.ECRupdate = function(locals, callback) {
```

Comment:
```suggestion
module.exports.ecrUpdate = function(locals, callback) {
```

We've been trying to standardize on strict camelCasing, including for words that are normally uppercase. I'd also be fine to go with `ECRUpdate`.